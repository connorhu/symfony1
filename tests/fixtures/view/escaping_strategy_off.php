<?php

require_once __DIR__.'/common.inc';

$context = sfContext::getInstance();
$dispatcher = $context->getEventDispatcher();

$parameterHolder = new sfViewParameterHolder(new sfEventDispatcher(), array('foo' => 'bar'));
$parameterHolder->setEscaping('off');
$values = $parameterHolder->toArray();

echo count($values);
echo "\n";
echo count($values['sf_data']);
echo "\n";
echo $values['foo'];
echo "\n";
echo $values['sf_data']['foo'];
