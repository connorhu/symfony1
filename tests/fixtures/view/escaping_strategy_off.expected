2
1
bar
bar